ElectronicObserverTypes/Extensions/ShipDataExtensions.cs

```diff
@@ -619,4 +619,12 @@ public static bool CanSink(this IShipData ship, IFleetData fleet)
 
 	private static bool HasDamecon(this IShipData ship) => ship.AllSlotInstance
 		.Any(e => e?.MasterEquipment.CategoryType is EquipmentTypes.DamageControl);
+
+	public static DamageState GetDamageState(this IShipData ship) => ship.HPRate switch
+	{
+		> 0.75 => DamageState.Healthy,
+		> 0.5 => DamageState.Light,
+		> 0.25 => DamageState.Medium,
+		_ => DamageState.Heavy,
```

Comment:
Shouldn't we also have a Damage state "Sunk" in case HP is 0 or below ?

Reply:
added